lib/package.gi

```diff
@@ -2270,6 +2270,14 @@ InstallGlobalFunction( ValidatePackageInfo, function( info )
     TestOption( record, "BannerString", IsString, "a string" );
     TestOption( record, "TestFile", IsFilename,
                 "a string denoting a relative path to a readable file" );
+    if not IsBound( record.TestFile ) then
```

Comment:
It would be more consistent with the style of the rest of the function to define something like `TestRecommended` 
which could uniformly provide feedback on aspects of a `PackageInfo.g`file which are recommended, but not mandatory. Optional arguments (or an info class) could be used to control whether this feedback was printed.

Reply:
`TestRecommended` is a good idea. For example, one could detect if the package is hosted on GitHub and in this case recommend to add fields for source code repository and issue tracker. I will revisit this.